klayout/python/klive_server.py

```diff
@@ -53,14 +55,17 @@ def new_connection(self):
                     line = connection.readLine()
                     data = json.loads(line)
                     # Interpret the data
-                    gds_path = data["gds"]
+                    if "gds" in data:
+                        gds_path = data["gds"]
+                    if "lyrdb" in data:
+                        lyrdb_path = data["lyrdb"]
```

Comment:
If you don't load a gds in this case, how do you find the correct view to show the (ly)rdb?

Reply:
My initial thought was to have a separate method that only loads the rdb.  But then I thought it is easiest to open the layout and the rdb at the same time, so the gds parameter is actually required.  Line 58-59 can be deleted.